ScorchApi/ScorchApiV2/Controllers/EntityController.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon;
+using Amazon.DynamoDBv2;
+using Microsoft.AspNetCore.Mvc;
+using Amazon.DynamoDBv2.DocumentModel;
+using Newtonsoft.Json;
+using ScorchApiV2.Models;
+
+namespace ScorchApiV2.Controllers
+{
+    [Route("api/[controller]")]
+    public class EntityController : Controller
+    {
+        private static AmazonDynamoDBClient _client = new AmazonDynamoDBClient(RegionEndpoint.USEast1);
+        private static string _entityTableName = "DnD-Entity";
+        private static Table _entityTable;
+
+        public EntityController()
+        {
+            _entityTable = Table.LoadTable(_client, _entityTableName);
+        }
+
+        [HttpGet("{entityId}")]
+        public async Task<Entity> GetEntity(Guid campaignId, int entityNumber)
```

Comment:
so this route will be /api/entity/{entityNumber}?campaignId={id}

is that what you want?